flaml/automl/model.py

```diff
@@ -2,36 +2,42 @@
 #  * Copyright (c) FLAML authors. All rights reserved.
 #  * Licensed under the MIT License. See LICENSE file in the
 #  * project root for license information.
+import logging
+import math
+import os
+import shutil
+import signal
+import sys
+import time
 from contextlib import contextmanager
 from functools import partial
-import signal
-import os
 from typing import Callable, List, Union
+
 import numpy as np
-import time
-import logging
-import shutil
-import sys
-import math
+

```

Comment:
Could we avoid these kind of changes? In fact, I like the new format, but would prefer use a new PR just for formatting.

Reply:
> Yes, I did not want to do it but pre-commit made all these changes. Maybe I configured it wrong. Should I try to revert it?

I hope so, if you can do that without too much trouble. I tried to clone the repo into a new folder and run precommit to all files, nothing changed. So I guess maybe you've different configs. Given this, next time a new PR is coming, I don't know if any changes in this PR will be changed back again (high chance not, but I'm not sure).

Reply:
what do you think on this format thing? @sonichi , @qingyun-wu

Reply:
@Borda  do you have any suggestion?

Reply:
Yep, I think changes were made by vscode-isort / isort library, I think it should be possible to revert it without too much of a hassle

Reply:
I think the format change is OK. We can keep them.

Reply:
I think it's doable to `isort` the whole library, add it to pre-commit hook with args `--profile black` and add it to dev dependencies.